composer.lock

```diff
@@ -44,16 +44,16 @@
         },
         {
             "name": "automattic/jetpack-constants",
-            "version": "v1.3.0",
+            "version": "v1.4.0",
```

Comment:
Can you please update the composer.lock only for the REST API package?

Reply:
Taking a look